rhizome/host/lib/vm_setup.rb

```diff
@@ -456,6 +456,7 @@ def cloudinit(unix_user, public_keys, gua, nics, swap_size_bytes, boot_image, dn
 server=2620:fe::fe
 server=2620:fe::9
```

Comment:
>  Add dns ipv4 to the dhcp payload and remove static dns conf 

I would add to this commit message. You can mention the provenance (you found them by inspection) even if they solved no acute problem you know of at the time, and I think you documented at least some small changes to output to commands.

Reply:
Do you like it now?